<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPRSeparateDetailTableWidget</class>
 <widget class="QWidget" name="SPRSeparateDetailTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="spTables">
     <property name="styleSheet">
      <string notr="true">QSplitter::handle:horizontal {
   width: 6px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(242, 242, 242), stop:1 rgb(149, 150, 149))
}

QSplitter::handle:vertical {
    height: 6px;
	background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(149, 150, 149), stop:1 rgb(242, 242, 242))
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="upWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="SPRSeparateDetailsTable" name="tSeparateDetail">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="downWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="SPRSeparateDetailsSummaryTable" name="tSeparateDetailSummary">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="threadsWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="gbThreads">
               <property name="title">
                <string>Ручьи</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>3</number>
                </property>
                <property name="leftMargin">
                 <number>9</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="checkBox">
                  <property name="text">
                   <string>Ручей 1</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_2">
                  <property name="text">
                   <string>Ручей 2</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_3">
                  <property name="text">
                   <string>Ручей 3</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_4">
                  <property name="text">
                   <string>Ручей 4</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="hsTreads">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QWidget" name="widget" native="true">
                  <layout class="QFormLayout" name="formLayout">
                   <property name="horizontalSpacing">
                    <number>3</number>
                   </property>
                   <property name="leftMargin">
                    <number>9</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Не учитывать камни со временем измерения меньше (мс)</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Время опроса (сек)</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QSpinBox" name="leMinimumTimeScope">
                     <property name="minimum">
                      <number>16</number>
                     </property>
                     <property name="maximum">
                      <number>99999</number>
                     </property>
                     <property name="singleStep">
                      <number>16</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QSpinBox" name="leCommandTimeInterval">
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="bSave">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bClear">
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bEndScope">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Закончить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bBeginScope">
            <property name="text">
             <string>Начать </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SPRSeparateDetailsTable</class>
   <extends>QTableView</extends>
   <header>sprseparatedetailstable.h</header>
  </customwidget>
  <customwidget>
   <class>SPRSeparateDetailsSummaryTable</class>
   <extends>QTableView</extends>
   <header>sprseparatedetailssummarytable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
